### DISCUSSION TITLE
I am getting a string '(array)' in the config instead of the expected value

### DISCUSSION THREAD

**Discussion by capaj** [prisma/prisma#9510] [Q&A] [answered]
in my prisma.schema file I have a generator like this:
```
generator typegraphql {
    provider = "typegraphql-prisma"
    output   = "../generated/typegraphql-prisma"
    emitOnly = ["enums", "models"]
}
```

problem is when I run it, I am getting this:
```
  generator: {
    name: 'typegraphql',
    provider: { fromEnvVar: null, value: 'typegraphql-prisma' },
    output: {
      value: '/home/capaj/work-repos/authier-repos/authier/backend/generated/typegraphql-prisma',
      fromEnvVar: null
    },
    config: { emitOnly: '(array)' }, // I am expecting ["enums", "models"] here, instead I am getting a string here 
    binaryTargets: [],
    previewFeatures: [],
    isCustomOutput: true
  },
```

I am expecting ["enums", "models"] here, instead I am getting a string in the config. Why?

**Comment by ryands17** [comment]
@capaj 👋 
This seems like an issue with the TypeGraphQL generator so I would suggest asking this in [TypeGraphQL](https://github.com/MichalLytek/type-graphql/discussions) itself 🙂

**Reply by capaj** [reply]
I am actually logging this right on the very first line of that generator.
How are you parsing the schema file?

**Reply by capaj** [reply]
This is the very first line of `onGenerate` callback being passed into `@prisma/generator-helper`.
![image](https://user-images.githubusercontent.com/1305378/135269431-03a091ac-2546-4f0f-acaa-826ff0c52811.png)

**Reply by janpio** [reply]
This might very well be a not yet supported feature. 
The generator API in general is pretty much experimental, so I would not be surprised if this is the case.

**Reply by capaj** [reply]
@janpio oh ok I'll use a string value for now, but supporting arrays would be great to have. I will create a feature request for this.

**Reply by zackdotcomputer** [reply, marked as answer]
If you, like me, wound up here from Google - here's a link to the Issue: https://github.com/prisma/prisma/issues/9511

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
